docs/topics/configmap-mission-intro.adoc

```diff
@@ -6,9 +6,9 @@ This mission shows you how to:
 * Deploy changes to the `ConfigMap` configuration of running applications.
 
 .About Externalized Configuration
-It is important for the application configuration to be externalized and separate from its code. This allows the application configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your code base and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
-Kubernetes increases the complexity of this by adding pods, or groups of containers representing a deployment, and polyglot environments.
-
+It is important for the application configuration to be externalized and separated from its code. This allows the application configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your code base and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
+Microservices architectures and polyglot environments add a layer of complexity to managing an applications configuration. Applications are comprised of independent services and the configuration is distributed.
```

Comment:
an applications configuration -> an application's configuration *or* applications configuration
Applications are comprised of independent, distributed services, each potentially with their own configuration.